src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesBackupContainer.cs

```diff
@@ -41,7 +41,7 @@ public class GetAzureRmRecoveryServicesBackupContainer : RSBackupVaultCmdletBase
         [Parameter(Mandatory = false, Position = 2,
             HelpMessage = ParamHelpMsgs.Container.BackupManagementType)]
         [ValidateNotNullOrEmpty]
-        [ValidateSet("AzureVM", "MARS", "AzureSQL", "AzureStorage")]
+        [ValidateSet("AzureVM", "MARS", "AzureWorkload", "AzureStorage")]
```

Comment:
sorry I didn't identify this one before but can you add "AzureSQL" back to the validation set?
it's breaking change to make validation set more exclusive.

It seems like you didn't introduce SDK changes which end up with calling different API version, why is this "AzureSQL" value not supported anymore?